tests/testing_config/custom_components/test/device_tracker.py

```diff
@@ -5,7 +5,7 @@
 from homeassistant.components.device_tracker.const import SOURCE_TYPE_ROUTER
 
 
-def get_scanner(hass, config):
+async def async_get_scanner(hass, config):
```

Comment:
Does this have to be renamed / made async? It's just a simple return.

Reply:
It's renamed to make it clear the device_tracker's get_scanner function is async safe